lib/compiler/src/v3_kernel.erl

```diff
@@ -909,14 +902,6 @@ make_vars(Vs) -> [ #k_var{name=V} || V <- Vs ].
 add_var_def(V, St) ->
     St#kern{ds=cerl_sets:add_element(V#k_var.name, St#kern.ds)}.
 
-%%add_vars_def(Vs, St) ->
-%%    Ds = foldl(fun (#k_var{name=V}, Ds) -> add_element(V, Ds) end,
-%%	       St#kern.ds, Vs),
-%%    St#kern{ds=Ds}.
-
-%% is_remote_bif(Mod, Name, Arity) -> true | false.
-%%  Test if function is really a BIF.
```

Comment:
It looks like this should've been kept, was it removed by accident?

Reply:
Fixed.